app/controllers/users_controller.rb

```diff
@@ -4,7 +4,10 @@ class UsersController < ApplicationController
   before_action :logged_in_user, only: %i(index)
   before_action :admin_user, only: %i(destroy)
```

Comment:
nếu chỉ khai báo cho 1 action thì em dùng `symbol` cho đơn giản nha:
```suggestion
  before_action :admin_user, only: :destroy
```

Reply:
em đang làm theo tutorial, họ đặt tên method là `admin_user`
tuy nhiên, theo phong cách của Ruby và Rails thì em có thể đặt tên khác phù hợp hơn là `verify_admin!` :)